test/OpenTelemetry.Tests/Metrics/AggregatorTest.cs

```diff
@@ -102,13 +109,17 @@ public void HistogramWithOnlySumCount()
 
             histogramPoint.TakeSnapshot(true);
 
+            var count = histogramPoint.GetHistogramCount();
+            var sum = histogramPoint.GetHistogramSum();
+
             // Sum of all recordings
-            Assert.Equal(40, histogramPoint.DoubleValue);
+            Assert.Equal(40, sum);
 
             // Count  = # of recordings
-            Assert.Equal(7, histogramPoint.LongValue);
-            Assert.Null(histogramPoint.BucketCounts);
-            Assert.Null(histogramPoint.ExplicitBounds);
+            Assert.Equal(7, count);
+
+            Assert.Throws<InvalidOperationException>(() => histogramPoint.GetBucketCounts());
```

Comment:
I think we should really evaluate if we want to throw in these getters, as this might be unexpected. Specially since now the behavior is changed.